mama/c_cpp/src/c/mama/publisher.h

```diff
@@ -196,8 +283,61 @@ extern mama_status
 mamaPublisher_getTransport (mamaPublisher   publisher,
                             mamaTransport*  transport);
 
+/**
```

Comment:
Good question, we were trying to make changing the structure easier, but most people will probably just put it on the stack and assign themselves. If you want I can remove them.